catalog-items.html

```diff
@@ -0,0 +1,371 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SKISIRS#4587 - BinaBox</title>
+    <link rel="stylesheet" href="./css/style.css">    
+</head>
+
+<body>   
+    <header class="page-header">           
+        <a href="index.html" class="logo">
+            <img src="./img/Logo.svg" alt="logo BinaBox" width="35" height="35" loading="lazy" class="logo-img">
+            <span class="logo-text">Binabox</span>
+        </a>           
+
+        <nav class="page-header-nav">
+            <ul class="header-nav-list">
+                <li class="nav-item"> 
+                    <a href="index.html" class="nav-dropbtn" >Home</a>
+
+                    <ul class="nav-dropdown-content">
+                        <a href="catalog.html" class="nav-link">Catalog</a>
+                        <a href="contact.html" class="nav-link">Contact us</a>
+                    </ul>
+                </li>
+
+                <li class="nav-item"> 
+                    <a href="index.html" class="nav-dropbtn" >About</a>
+
+                    <ul class="nav-dropdown-content">
+                        <a href="catalog.html" class="nav-link">Lorem.</a>
+                        <a href="contact.html" class="nav-link">Lorem.</a>
+                    </ul>
+                </li>
+            <!-- and so on -->
+            </ul>
+        </nav>
+       
+        <button class="btn-theme-toggle" onclick="toggleTheme()">
+            <img src="./img/switch-sun.svg" alt="sun" width="20" height="20" loading="lazy" class="theme-toggle-img">                   
+        </button>
+       
+
+        <div class="page-header-connect">
+            <a href="https://discord.com" target="_blank" class="link-discord">  
+                <img src="./img/Discord.svg" alt="discord" width="20" height="20" loading="lazy" class="logo-discord">
+                Discord              
+            </a>
+
+            <a href="https://connect-wallet.html" target="_blank" class="link-connect-wallet">
+                <img src="./img/wallet.svg" alt="wallet" width="20" height="20" loading="lazy" class="logo-wallet">
+                Connect
+            </a>                              
+        </div>
+    </header>
+    
+    <main class="content">
+        <section class="item-detail">
+            <header class="item-detail-heading">  <!-- если header уже имеет класс, есть ли смысл прописывать класс h, если кроме него там ничего нет? -->
+                <h1>ITEM DETAIL</h1> 
+            </header>
+           
+            <nav class="item-detail-nav">
+                <ul class="item-detail-navlist hidden">
+                    <li class="item-detail-navitem">
+                        <a href="index.html" class="item-detail-navlink">Home</a>                       
+                    </li>
+
+                    <div class="vertical-line">&VerticalLine;</div> <!-- <hr class="vertical-line"> - этот вариант не очень или на уровне с этим? -->
```

Comment:
it's unnecessary element, because it's empty 
so we were talk, that we dont' use empty elements, because it can be replaced by css styles